src/App.js

```diff
@@ -22,34 +19,31 @@ function App() {
 
   // if (error) console.log(error);
   // for Get user------------------------------------------>
-
-  useEffect(()=>{
-    getData();
-  }, [])
   const [data, setData] = useState([]);
-  const [loading, setLoading]=useState(false);
-
-   const getData = async ()=>{
-
+  const [loading, setLoading] = useState(false);
 
+   const getData = async () => {
     try{
      const res = await axios.get("https://randomuser.me/api?results=1")
      setData(res.data.results)
      setLoading(true)
-    }catch(err){
-      console.log(err.message)
+    } catch(err) {
+      console.error(err.message)
```

Comment:
Should there also be an error state? Like what happens when/if this fetch failed?

Reply:
Okay nice catch! (no pun intended) lol